bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRIssueWorkItem.java

```diff
@@ -101,14 +101,14 @@ public Collection<WorkItem> run(Path scratchPath) {
                 .filter(Issue::isOpen)
                 // This will mix time stamps from the IssueTracker and the Forge hosting PRs, but it's the
                 // best we can do.
-                .map(pr -> new PullRequestWorkItem(pr.repository(), pr.id(), csrIssue.project(), csrIssue.updatedAt(), errorHandler))
+                .map(pr -> new CheckWorkItem(bot.getPRBot(pr.repository().name()), pr.id(), errorHandler, csrIssue.updatedAt(), true, true))
```

Comment:
We should probably only schedule this if the PullRequestBot in question is configured to handle CSR.

Reply:
Good idea!